.github/workflows/main.yml

```diff
@@ -2,8 +2,67 @@ name: CI
 on: [push, pull_request]
 
 jobs:
-  test:
-    name: Test
+  buildlibc:
+    name: Build libc
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+
+    - name: Install clang (Windows)
+      shell: bash
+      run: |
+          curl -fsSLO https://releases.llvm.org/8.0.0/LLVM-8.0.0-win64.exe
+          7z x LLVM-8.0.0-win64.exe -y -o"llvm"
+          echo "::add-path::$(pwd)/llvm/bin"
+          echo "::set-env name=WASM_AR::$(pwd)/llvm/bin/llvm-ar.exe"
+      if: matrix.os == 'windows-latest'
+
+    - name: Install llvm-nm (Windows)
+      shell: bash
+      run: |
+        # Add --no-self-update as a workaround for
+        # https://github.com/microsoft/azure-pipelines-image-generation/issues/1224
```

Comment:
Looks like this issue is closed.. can we drop this step completely?

Reply:
fixed d0ea5ee